File: atividade-3/atividade.go
package main

import "fmt"

type ContaBancaria struct {
	titular string
	saldo   float64
}

func (c *ContaBancaria) Depositar(valor float64) {
	c.saldo += valor
}

func (c *ContaBancaria) Sacar(valor float64) {
	if valor <= c.saldo {
		c.saldo -= valor
	} else {
		fmt.Println("Saldo insuficiente")
	}
}

func (c ContaBancaria) ExibirSaldo() {
	fmt.Printf("Saldo de %s: %.2f\n", c.titular, c.saldo)
}

func main() {
	conta := ContaBancaria{titular: "João", saldo: 1000}
	conta.Depositar(500)
	conta.Sacar(200)
	conta.ExibirSaldo() 
}

File: atividade-3/atividade.py
class ContaBancaria:
    def __init__(self, titular, saldo=0):
        self.__saldo = saldo  
        self.titular = titular

    def depositar(self, valor):
        self.__saldo += valor

    def sacar(self, valor):
        if valor <= self.__saldo:
            self.__saldo -= valor
        else:
            print("Saldo insuficiente")

    def exibir_saldo(self):
        print(f"Saldo de {self.titular}: {self.__saldo}")

conta = ContaBancaria("João", 1000)
conta.depositar(500)
conta.sacar(200)
conta.exibir_saldo() 